### ISSUE TITLE
Placing Blueprint into Engineer's Workbench causes client to crash

### ISSUE THREAD

**Issue by imNel** [FTBTeam/FTB-Modpack-Issues#4514]
### Modpack

FTB Inferno

### Modpack version

1.10.0

### Log Files

https://mclo.gs/hfw3rUL

### Describe the bug

Placing Blueprint into Engineer's Workbench causes client to crash. Server is fine but any client that tries to render the block crashes. Same issue after disabling embedium/oculus.

### Steps to reproduce

Add a blueprint to the workbench.

### Expected behaviour

Be able to use the workbench

**Comment by aofekiko** [comment]
I've also encountered this error.

**Comment by bgoehringer** [comment]
I encountered this bug as well and was able to fix it by downgrading the installed version of Immersive Engineering for the modpack.

If you feel adventurous, downgrading it to ImmersiveEngineering-1.18.2-8.3.1-157 worked for me. The blueprint graphic is bugged slightly on the table but at least it us usable and unblocked my progression.

Make sure to back up your world before mucking with anything though!

**Comment by Drackmord** [comment]
Go to config, open immersiveenginering-client, set enableVBO = true

**Comment by joeyjumper94** [comment]
> Go to config, open immersiveenginering-client, set enableVBO = true

that did not work for me.

**Comment by Gaz492** [comment]
> > Go to config, open immersiveenginering-client, set enableVBO = true
> 
> that did not work for me.

Would you be able to post the crash report you ran into?

Unfortunately the mod author no longer supports the minecraft version that Inferno runs on so its unlikely that this issue will resolved in a mod update.

We may look into downgrading the mod as it may resolve the issue as mentioned in another issue.

> #4514 Link to curseforge's download of the downgrade: [curseforge.com/minecraft/mc-mods/immersive-engineering/files/4369369](https://www.curseforge.com/minecraft/mc-mods/immersive-engineering/files/4369369) Downgrading the immersive engineering mod works apparently
